templates/common/_base/files/nm-ignore-sdn.yaml

```diff
@@ -4,5 +4,5 @@ contents:
   inline: |
     # ignore known SDN-managed devices
     [device]
-    match-device=interface-name:br-int;interface-name:br-local;interface-name:br-nexthop,interface-name:ovn-k8s-*,interface-name:k8s-*;interface-name:tun0;interface-name:br0;driver:veth
+    match-device=interface-name:br-int;interface-name:patch-br-int-*,interface-name:br-local;interface-name:br-nexthop,interface-name:ovn-k8s-*,interface-name:k8s-*;interface-name:tun0;interface-name:br0;driver:veth
```

Comment:
Not to be a broken record but...in the future can we do something like have the SDN daemonset write this out dynamically or so?

Or perhaps even better, something like extended attributes on network interfaces that NM could react to?

(I think this came up in e.g. https://bugzilla.gnome.org/show_bug.cgi?id=731014 too before)

Reply:
The SDN is agnostic of NM. 
I did not know extended attributes on network interfaces was a thing, but we would need support from NM as well or is it there already?
I agree with general idea to try to remove as much as we can from here, but we are bound to have some platform related bits here and there.